http_service/bugbug_http/models.py

```diff
@@ -121,15 +121,15 @@ def schedule_tests(branch: str, rev: str) -> str:
         revs = get_hgmo_stack(branch, rev)
     except requests.exceptions.RequestException:
         LOGGER.warning(f"Push not found for {branch} @ {rev}!")
-        return "NOK"
+        raise
```

Comment:
In this case we should keep returning "NOK", if the push can't be found it's not useful to retry

Reply:
This was left unstaged. Fixed now.